openimis.json

```diff
@@ -66,22 +62,10 @@
       "name": "CalculationModule",
       "npm": "@openimis/fe-calculation@git+https://github.com/openimis/openimis-fe-calculation_js.git#develop"
     },
-    {
-      "name": "PolicyHolderModule",
-      "npm": "@openimis/fe-policyholder@git+https://github.com/openimis/openimis-fe-policyholder_js.git#develop"
-    },
-    {
-      "name": "ContributionPlanModule",
```

Comment:
@Shahzaibahmad97 ContributionPlan module cannot be removed due to containing PaymentPlan (used in social protection payment flow too) mechanism inside this module. 
@delcroip FYI maybe we can distinguish this modules to have separate one for PaymentPlan and for ContributionPlan, what do you think?

Reply:
@sniedzielski rest of the modules can be removed?

Reply:
I think yes, the only module which cannot be removed is 'ContributionPlan'. I'll check also on my computer to double check this.

Reply:
I'll wait for it then. As when discussing with Andrea, he wasn't sure what this contribution plan part does and therefore we wanted to remove this or hide this through permissions at least.

Reply:
Yes, maybe the easiest way is to do in the same way as you did for health facility in previous PRs. Based on permission - ContributionPlan can be hided. 

The second approach is to move PaymentPlan to separarate module - it could take more time, PaymentPlan is also used in Claim BatchRun - therefore some regression tests might be required to ensure nothing is broken after this potentlal change. 

@delcroip Let me know what do you think about two strategies how to hide ContributionPlan.